### PR TITLE
remove @author tags from (mainly) src/library

### PR DESCRIPTION
**Pull request by SethTisue**
I'm not pushing hard for this to be merged. I'm just suggesting it

why do it?
* the credits are bitrotted; most credits date back to very very early days; it's pretty random whether any given file has these credits at all, and if so, how much it still corresponds with any concept of "real" authorship
* authorship of the standard library is collective, code "ownership" is discouraged

and anyway, actual authorship information is available in the Git history.

### CODE REVIEW

File: src/library/scala/util/Either.scala [hunk 1]
```diff
@@ -526,7 +521,6 @@ object Either {
 
   /** Projects an `Either` into a `Left`.
    *
-   *  @author <a href="mailto:research@workingmouse.com">Tony Morris</a>, Workingmouse
```
Comment (hrhino) [inline src/library/scala/util/Either.scala:529]: If this doesn't get merged, workingmouse.com is [only $9,888](http://workingmouse.com)!

File: src/library/scala/runtime/BoxesRunTime.java [hunk 2]
```diff
@@ -24,9 +24,6 @@
   *     be a boxed value.
   *   - Standard value operators for boxed number and quasi-number values.
   *
-  * @author  Gilles Dubochet
-  * @author  Martin Odersky
-  * @contributor Stepan Koltsov
```
Comment (hrhino) [inline src/library/scala/runtime/BoxesRunTime.java:29]: This needs an update in the `instrumented` version.

**Review by hrhino — COMMENTED** [review]
I'm not an `@author`, so maybe my opinion doesn't matter, but :+1: anyways.

**Comment by milessabin** [comment]
Blimey! My name is on `ControlThrowable`!?! That must have been one of my dark periods ...

**Comment by NthPortal** [comment]
The only advantage I see of having `@author` tags is that you know who to ping if there's a bug, and since they are apparently wildly inaccurate and/or outdated... ¯\\\_(ツ)_/¯

**Comment by Ichoran** [comment]
I think `git blame` is the appropriate way to figure these things out, so I support getting rid of the author tags as distractions from the way to actually find out who did what.

**Pull request merged**